hermes_core/util/util.py

```diff
@@ -76,20 +76,10 @@ def create_science_filename(
             "The underscore symbol _ is not allowed in mode or descriptor."
         )
 
-    filename = (
-        "hermes_{inst}_{mode}_{level}{test}_{descriptor}_{time}_v{version}".format(
-            inst=hermes_core.INST_TO_SHORTNAME[instrument],
-            mode=mode,
-            level=level,
-            test=test_str,
-            descriptor=descriptor,
-            time=time_str,
-            version=version,
-        )
-    )
+    filename = f"hermes_{hermes_core.INST_TO_SHORTNAME[instrument]}_{mode}_{level}{test_str}_{descriptor}_{time_str}_v{version}"
     filename = filename.replace("__", "_")  # reformat if mode or descriptor not given
 
-    return filename
+    return filename + ".cdf"
```

Comment:
**Nitpick**: Would it be better to include the extension with the filename? Or maybe even create an extension variable so it would be something like: `return filename + file_extension`, that way the `.cdf` isn't always hardcoded and we could use it elsewhere if needed in the future.

Reply:
Agreed and fixed.